src/cfnlint/customRules/Operators.py

```diff
@@ -2,17 +2,42 @@
 Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 SPDX-License-Identifier: MIT-0
 """
+# pylint: disable=W0622
+from cfnlint.rules import Match
 
 
-def equalsOp(rule, propertyList):
+def equalsOp(template, rule, propertyList):
     """ Process EQUALS operators """
+
     if not propertyList:
-        return 'EQUALS'  # Resource type not found
+        return Match(
+            1, 1,
+            1, 1,
+            template.filename, CustomRule('E9200'),
+            rule.operator, None)  # Resource type not found
```

Comment:
[still think we can just get rid of these](https://github.com/aws-cloudformation/cfn-python-lint/pull/1559#discussion_r434928852)

think the general expectation of property type constraints will be that they're applied where applicable and ignored otherwise

Reply:
+1, "# Resource type not found" is just a fact of the template, not an error

Reply:
I appear to have rolledback the fix there. Repatched in the new update!